CHARTER.adoc

```diff
@@ -0,0 +1,195 @@
+
+
+
+== Business Value Proposition
+
+In the geospatial world, new cloud native data formats are emerging. Zarr is a generic data format for n-dimensional arrays that enables access to data in compressed chunks of the original array and has become increasingly popular to use for geospatial purposes. In June 2022, the OGC endorsed a community standard of Zarr V2.0 (https://zarr.readthedocs.io/en/stable/spec/v2.html). The purpose of this charter is to adopt a more explicit GeoZarr as an OGC Standard which would provide guidance to standardize the approach for encoding various aspects of geospatial data in zarrs.
+
+== Scope of Work
+
+The goal of the GeoZarr specification is to establish flexible and inclusive conventions for the Zarr cloud-native format, specifically designed to meet the diverse requirements within the geospatial domain. These conventions aim to provide a clear and standardized framework for organizing and describing data, ensuring unambiguous representation. 
+
+The objectives of GeoZarr conventions includes:
+
+1. Compatibility: Ensuring easy compatibility with popular mapping and data analysis tools such as GDAL, Xarray, ArcGIS, and QGIS, enabling seamless integration into existing workflows.
+2. Visualisation: Simplifying the creation and display of geospatial data in web browsers without the need for complex workarounds, making geospatial information more accessible to users.
+3. Dimensions: Supporting multidimensional data, such as hyperspectral and altitude information, to address diverse geospatial data requirements.
+4. Data Discovery: Providing metadata for discovering, accessing, and retrieving the data, including subsets or different arrangements of the data.
+5. Mixing Data: Facilitating the combination of different types of geospatial data, including satellite images, elevation maps, and weather models, to create comprehensive and informative datasets.
+8. Flexibilty: Allowing scientists and researchers to work with diverse data types and projections in their preferred software and programming languages, promoting flexibility and adaptability in geospatial data processing and analysis.
+
+Specifically, the convention should provide guidance to standardize the approach for encoding various aspects of geospatial data, including:
+
+* Multiple related variables with heterogeneous coordinates (e.g., children or linked datasets)
+* Multiple resolutions of the data, possibly leveraging multiscale drafts
+* Data subsets that are only available at certain resolutions
+* Multiple projections
+* Multi-dimensional optimizations (rechunking)
+* Supporting typical Earth observation (EO) products (for example, how to encode multispectral bands)
+* Encoding of time series that have not been normalized
```

Comment:
I don't understand what is meant here either. This feels too specific to put in the charter.
```suggestion
```

Reply:
Not exactly. This covers the very usual use case of series of observations for which the footprint/pixels are not aligned but covering the same area. 
So can we provide a convention to encode this as a single array (with time dimension) and how (and let know the reader).

Reply:
Consider expanding the definition to "Encoding time series of images that share a footprint but not grid." 

That said, this is a very boutique data format that may not be worth the energy to standardize in a single ZARR structure.

Reply:
I am also in favor of removing, i am not sure we can generalize any convention enough to cover all use cases. It is indeed very common, but you often have additional dimensions that allow representation as a single array. Lots of level 2 products do this.

Reply:
Being not sure... means this question should be addressed by the WG and thus, present in the charter.  :)